pyvista/plotting/renderer.py

```diff
@@ -1249,6 +1255,9 @@ def show_bounds(
         # set bounds
         if bounds is None:
             bounds = np.array(mesh.GetBounds())
```

Comment:
I wonder why this didn't do this instead:
```suggestion
            bounds = np.array(mesh.bounds)
```